biome.json

```diff
@@ -13,7 +13,13 @@
     "linter": {
       "enabled": true,
       "rules": {
-        "recommended": true
+        "recommended": true,
+        "correctness": {
+          "noInvalidUseBeforeDeclaration": "off"
```

Comment:
This lint has disabled because of the pattern used in the test files with declare global, etc